src/main/webapp/WEB-INF/jsp/admin/deleteEventCRF.jsp

```diff
@@ -17,10 +17,10 @@
 <c:set var="dteFormat"><fmt:message key="date_format_string" bundle="${resformat}"/></c:set>
 
 <h1><span class="title_manage">
-<fmt:message key="delete_CRF_from_event" bundle="${resword}"/>
```

Comment:
can you delete the translations for these removed keys from words.properties and notes.properties?

Reply:
@svadla-oc sure, updated